packages/actor-query-operation-leftjoin/test/ActorQueryOperationLeftJoin-test.ts

```diff
@@ -124,6 +124,82 @@ describe('ActorQueryOperationLeftJoin', () => {
       });
     });
 
+    it('should correctly handle left hand bindings that are missing the variables of the expression', async() => {
+      const expression = {
+        expressionType: 'term',
+        term: DF.literal('nonemptystring'),
+        type: 'expression',
+      };
+
+      mediatorQueryOperation = {
+        mediate: (arg: any) => Promise.resolve({
+          bindingsStream: new ArrayIterator([
+            BF.bindings([[ DF.variable('c'), DF.literal('1') ]]),
+            BF.bindings([[ DF.variable('a'), DF.literal('1') ]]),
+          ], { autoStart: false }),
+          metadata: () => Promise.resolve({ cardinality: 3, canContainUndefs: true, variables: [ DF.variable('a') ]}),
+          operated: arg,
+          type: 'bindings',
+        }),
+      };
+
+      actor = new ActorQueryOperationLeftJoin({ name: 'actor', bus, mediatorQueryOperation, mediatorJoin });
```

Comment:
Making a new actor is not needed if you just override `mediatorQueryOperation.mediate` instead of creating a new `mediatorQueryOperation` object.